# Copyright 2008 Sun Microsystems, Inc.  All rights reserved.
# Use subject to license terms.
#
# Permission is hereby granted, free of charge, to any person obtaining a
# copy of this software and associated documentation files (the
# "Software"), to deal in the Software without restriction, including
# without limitation the rights to use, copy, modify, merge, publish,
# distribute, and/or sell copies of the Software, and to permit persons
# to whom the Software is furnished to do so, provided that the above
# copyright notice(s) and this permission notice appear in all copies of
# the Software and that both the above copyright notice(s) and this
# permission notice appear in supporting documentation.
# 
# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
# OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT
# OF THIRD PARTY RIGHTS. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
# HOLDERS INCLUDED IN THIS NOTICE BE LIABLE FOR ANY CLAIM, OR ANY SPECIAL
# INDIRECT OR CONSEQUENTIAL DAMAGES, OR ANY DAMAGES WHATSOEVER RESULTING
# FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT,
# NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION
# WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
# 
# Except as contained in this notice, the name of a copyright holder
# shall not be used in advertising or otherwise to promote the sale, use
# or other dealings in this Software without prior written authorization
# of the copyright holder.

Adds support for Sun's Frame Buffer Power Management (fbpm) extension

diff -urp -x '*~' -x '*.orig' xset.man xset.man
--- xset.man	2008-03-06 14:28:40.000000000 -0800
+++ xset.man	2008-03-19 10:00:11.190603000 -0700
@@ -1,5 +1,6 @@
 .\" $Xorg: xset.man,v 1.4 2001/02/09 02:05:59 xorgcvs Exp $
 .\" Copyright 1988, 1998  The Open Group
+.\" Copyright (c) 1997, 1999  Sun Microsystems, Inc.
 .\" 
 .\" Permission to use, copy, modify, distribute, and sell this software and its
 .\" documentation for any purpose is hereby granted without fee, provided that
@@ -43,6 +44,10 @@ xset - user preference utility for X
 	[dpms \fIstandby\fP [\fI suspend\fP [\fI off\fP]]] 
 	[dpms force {standby|suspend|off|on}]
 .br
+[+fbpm] [-fbpm]
+.br
+	[fbpm force {standby|suspend|off|on}]
+.br
 [fp=\fIpathlist\fP] 
 [-fp=\fIpathlist\fP] 
 [+fp=\fIpathlist\fP] 
@@ -124,7 +129,9 @@ the volume to the nearest value that the
 The \fB\-dpms\fP option disables DPMS (Energy Star) features.
 .TP 8
 .B +dpms
-The \fB+dpms\fP option enables DPMS (Energy Star) features.
+Enable Energy Star mode. Default times are standby = 1440, suspend = 1620, and
+off = 1800 seconds.
+.PP
 .TP 8
 .B dpms \fIflags...\fP
 The \fBdpms\fP option allows the DPMS (Energy Star) parameters to be
@@ -139,6 +146,32 @@ The first value given is for the `standb
 `suspend' mode, and the third is for the `off' mode.  Setting these
 values implicitly enables the DPMS features.  A value of zero disables
 a particular mode.
+Note that standby is less than suspend is less than off, if this condition is not met, an error is produced.
+.PP
+.TP 8
+.B +fbpm
+Enable frame buffer power management. NOTE: FBPM timing is linked to DPMS, See DPMS for default timeout values.
+.PP
+.TP 8
+.B -fbpm
+Disable frame buffer power management. NOTE: Does not affect DPMS timeouts.
+.PP
+.TP 8
+.B fbpm force standby
+Force to standby power level.
+.PP
+.TP 8
+.B fbpm force suspend
+Force to suspend power level.
+.PP
+.TP 8
+.B fbpm force off
+Force power to off.
+.PP
+.TP 8
+.B fbpm force on
+Force power to on.
+.PP
 .TP 8
 .B fp= \fIpath,...\fP
 The \fBfp=\fP sets the font path to the entries given in the path argument.
